### DISCUSSION TITLE
Events and cycles

### DISCUSSION THREAD

**Discussion by unicomp21** [esfx/esfx#90] [General]
@rbuckton is there an easy way to detect cycles at runtime when using esfx events?  Or am I thinking about this wrong?

**Comment by rbuckton** [comment]
As in one event triggering another event that in turn re-triggers the first? No, as this is a fairly low level primitive I generally expect that cycles like this will be handled by the user.

**Comment by unicomp21** [comment]
Yep, exactly what I was wondering, thanks!
